src/routes/+page.svelte

```diff
@@ -96,15 +78,9 @@
 			placeholder="Enter your message..."
 			class="m-auto outline-none active:border-none rounded-md p-2 flex-1"
 		/>
-		{#if $completionState == CompletionState.Loading}
-			<button on:click={stop}>
-				<Icon src={Stop} class="w-6 h-6 text-gray-500 hover:text-gray-950" />
-			</button>
-		{:else}
-			<button on:click={send}>
```

Comment:
We should still be streaming here. If we want to remove it from this app I would keep it in the markdown pr because thats the reason for it. But we can stream agents in langchain as far as I understand, they let you stream only the final message.

Reply:
Why do we want to use stream?

Reply:
Streaming is a lot better on the user side, it lets you see if you like what its completing and decide if you want to stop and send a different message. It just makes the user experience overall better.